ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -102,7 +102,19 @@ def get_connection_url(self):
         url += f"{self.host_port}"
         if self.database:
             url += f"/{self.database}"
-        logger.info(url)
+        if self.options:
+            url = self.get_options(options=self.options, url=url)
+
+        logger.debug(url)
+        return url
+
+    def get_options(self, options, url):
```

Comment:
are we capturing options in SQLSourceConfig, i.e user should be able to pass options via pipeline config in json

Reply:
yes, User will be able to pass the options in dictionary format via config files